### PR TITLE
Digger: Change manufacturer

### PR DESCRIPTION
**Pull request by MASHinfo**
Changed the manufacturer of Digger (1980) to "Gremlin / Sega".
Found the informations in the Digger rom 691.u20:

WRITTEN BY DAVID L. EVANS, 4535 TAFT AVENUE, LA MESA, CALIF. 92041 AND BEAT OUT ON THE KEYBOARD OF A NOVAL 765!
THANK YOU TERRY, BARBARA, CRAIG AND LARRY FOR SOME DEBUGGED SUBROUTINES...JUNE 25, 1980 GREMLIN/SEGA INC

Also Wikipedia says Gremlin/Sega: https://en.wikipedia.org/wiki/Heiankyo_Alien

Arcade versions
In 1980, a North American version titled Digger was created by Sega and Gremlin Industries, with minor changes in appearance.

### CODE REVIEW

File: src/mame/drivers/midyunit.cpp [hunk 1]
```diff
@@ -3063,9 +3063,9 @@ ROM_END
  *
  *************************************/
 
-GAME( 1988, narc,       0,        zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "Narc (rev 7.00)", MACHINE_SUPPORTS_SAVE )
-GAME( 1988, narc3,      narc,     zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "Narc (rev 3.20)", MACHINE_SUPPORTS_SAVE )
-GAME( 1988, narc2,      narc,     zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "Narc (rev 2.00)", MACHINE_SUPPORTS_SAVE )
+GAME( 1988, narc,       0,        zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "NARC (rev 7.00)", MACHINE_SUPPORTS_SAVE )
+GAME( 1988, narc3,      narc,     zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "NARC (rev 3.20)", MACHINE_SUPPORTS_SAVE )
+GAME( 1988, narc2,      narc,     zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "NARC (rev 2.00)", MACHINE_SUPPORTS_SAVE )
```
Comment (cuavas) [inline src/mame/drivers/midyunit.cpp]: What are you basing this on?  As far as I can tell, it's an abbreviation for “narcotics”, not an initialism, and hence shouldn't be capitalised.  The Wikipedia page seems to agree with me: https://en.wikipedia.org/wiki/Narc_(video_game)

File: src/mame/drivers/vicdual.cpp [hunk 2]
```diff
@@ -3909,7 +3909,7 @@ GAME( 1981, brdrlins,   brdrline, brdrline,  brdrline,  vicdual_state, empty_ini
 GAME( 1981, brdrlinb,   brdrline, brdrline,  brdrline,  vicdual_state, empty_init, ROT270, "bootleg (Karateco)", "Borderline (Karateco bootleg)", MACHINE_NO_SOUND | MACHINE_SUPPORTS_SAVE )
 GAME( 1981, brdrlinet,  brdrline, tranqgun,  tranqgun,  vicdual_state, empty_init, ROT270, "Sega", "Borderline (Tranquillizer Gun conversion)", MACHINE_NOT_WORKING | MACHINE_IMPERFECT_SOUND | MACHINE_SUPPORTS_SAVE ) // official factory conversion
 GAME( 198?, startrks,   0,        headons,   headons,   vicdual_state, empty_init, ROT0,   "bootleg (Sidam)", "Star Trek (Head On hardware)", MACHINE_IMPERFECT_SOUND | MACHINE_SUPPORTS_SAVE )
-GAME( 1980, digger,     0,        digger,    digger,    vicdual_state, empty_init, ROT270, "Sega", "Digger", MACHINE_NO_SOUND | MACHINE_SUPPORTS_SAVE )
+GAME( 1980, digger,     0,        digger,    digger,    vicdual_state, empty_init, ROT270, "Gremlin / Sega", "Digger", MACHINE_NO_SOUND | MACHINE_SUPPORTS_SAVE )
```
Comment (cuavas) [inline src/mame/drivers/vicdual.cpp]: If this was a work-for-hire and only sold by Sega with Sega branding, we wouldn't list the developer here.  Is there any evidence this was actually marketed as a Gremlin product?

Comment (MASHinfo) [reply src/mame/drivers/vicdual.cpp]: Is there any evidence this was actually marketed as a Gremlin product?
Gremlin / Sega is shown on the cabinet:
https://www.arcade-museum.com/game_detail.php?game_id=7583

File: src/mame/drivers/midyunit.cpp (hunk 1 above)
Comment (MASHinfo) [reply src/mame/drivers/midyunit.cpp]: OK. You can close this report. And i set up a new pull request with
Digger (Gremlin/Sega).

**Pull request closed without merging**